numpy/add_newdocs.py

```diff
@@ -463,6 +463,45 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
+add_newdoc('numpy.core', 'nested_iters',
+    """
+    Convenience function to create multiple nditers from a single ndarray.
+
+    Parameters (see nditer)
+    ---------
+    op : ndarray
+        The array to iterate over.
+
+    axes :  sequence of sequences
+        As in nditer "op_axes"
+
+    flags, op_flags, op_dtypes, order, casting, buffersize (optional): see nditer
+        parameters of the same name
```

Comment:
Strange formatting here too. Omit the colon, place the `see nditer` on a newline, and surround it with `` ` `` so that it creates a link.